src/libraries/Hooks.sol

```diff
@@ -15,18 +16,28 @@ import {Locker} from "./Locker.sol";
 library Hooks {
     using FeeLibrary for uint24;
     using Hooks for IHooks;
+    using SafeCast for int256;
 
     uint256 internal constant BEFORE_INITIALIZE_FLAG = 1 << 159;
     uint256 internal constant AFTER_INITIALIZE_FLAG = 1 << 158;
-    uint256 internal constant BEFORE_ADD_LIQUIDITY_FLAG = 1 << 157;
-    uint256 internal constant AFTER_ADD_LIQUIDITY_FLAG = 1 << 156;
-    uint256 internal constant BEFORE_REMOVE_LIQUIDITY_FLAG = 1 << 155;
-    uint256 internal constant AFTER_REMOVE_LIQUIDITY_FLAG = 1 << 154;
+
+    uint256 internal constant BEFORE_ADD_LIQ_FLAG = 1 << 157;
```

Comment:
not a fan of shortening LIQUIDITY to LIQ 🤔

Reply:
yeeah i can change it back, it was just pushing so many things onto lots of lines for linting and it was annoying me 😆